sources/AutoFilerTab.cpp

```diff
@@ -39,8 +46,8 @@ AutoFilerTab::AutoFilerTab()
 	BMessage panelMsg(MSG_FOLDER_CHOSEN);
 	fFilePanel = new BFilePanel(B_OPEN_PANEL, NULL, NULL,
 		B_DIRECTORY_NODE, true, &panelMsg, fRefFilter);
-	fFilePanel->Window()->SetTitle("AutoFiler: Add folders");
-	fFilePanel->SetButtonLabel(B_DEFAULT_BUTTON, "Add");
+	fFilePanel->Window()->SetTitle(B_TRANSLATE("AutoFiler: Add folders"));
+	fFilePanel->SetButtonLabel(B_DEFAULT_BUTTON, B_TRANSLATE("Add"));
```

Comment:
Definitely. Still, esp. when such a one-word string appears in several locations, it helps if you know where each appears. The B_TRANSLATE_CONTEXT can help, but we have to make it as easy as possible. Many translators aren't programmers and are not able to compile a test version.